MapboxNavigation/NavigationViewController.swift

```diff
@@ -163,6 +163,17 @@ public protocol NavigationViewControllerDelegate {
      Returns the center point of the user course view in screen coordinates relative to the map view.
      */
     @objc optional func navigationViewController(_ navigationViewController: NavigationViewController, mapViewUserAnchorPoint mapView: NavigationMapView) -> CGPoint
+    
+    /**
+     Called when a location has been discarded for being inaccurate.
+     
+     See `CLLocation.isQualified` for more information about what qualifies a location.
```

Comment:
Is this enough context for the developer to use the method? Or do we need to provide some sort of reason indicating why it’s a candidate for discarding? Without providing a reason, I think the developer only has a choice between discarding all the unqualified locations or including them all, in which case a property would be more convenient than a delegate method.

Reply:
What would you suggest, some sort of enum for each heuristic we investigate?